site/posts/sample-post.md

```diff
@@ -2,7 +2,7 @@
 title: "Sample Post"
 author: Me
 date: Jan 1, 2019
-tags: [slick, site]
+tags: [{ tag: slick }, { tag: site }]
```

Comment:
Any reason for the extra nesting? Seems unnecessary 😋

Reply:
I use records for tags because it allows me to get a handle on them in mustache loop. Are you able to just use an item at an index in a mustache loop?

Reply:
You can use `{{.}}` to refer to the thing you're iterating over:
https://stackoverflow.com/questions/6516297/can-mustache-iterate-a-top-level-array